plugins/GSdx/CMakeLists.txt

```diff
@@ -198,64 +159,103 @@ set(GSdxHeaders
     PSX/GPUSetupPrimCodeGenerator.h
     PSX/GPUState.h
     PSX/GPUVertex.h
-    stdafx.h
+    Renderers/Common/GSDevice.h
+    Renderers/Common/GSDirtyRect.h
+    Renderers/Common/GSFastList.h
+    Renderers/Common/GSFunctionMap.h
+    Renderers/Common/GSOsdManager.h
+    Renderers/Common/GSRenderer.h
+    Renderers/Common/GSTextureCache.h
+    Renderers/Common/GSTexture.h
+    Renderers/Common/GSVertex.h
+    Renderers/Common/GSVertexList.h
+    Renderers/Common/GSVertexTrace.h
+    Renderers/Null/GSDeviceNull.h
+    Renderers/Null/GSRendererNull.h
+    Renderers/Null/GSTextureNull.h
+    Renderers/HW/GSRendererHW.h
+    Renderers/HW/GSVertexHW.h
+    Renderers/SW/GSDrawScanlineCodeGenerator.h
+    Renderers/SW/GSDrawScanline.h
+    Renderers/SW/GSRasterizer.h
+    Renderers/SW/GSRendererSW.h
+    Renderers/SW/GSScanlineEnvironment.h
+    Renderers/SW/GSSetupPrimCodeGenerator.h
+    Renderers/SW/GSTextureCacheSW.h
+    Renderers/SW/GSTextureSW.h
+    Renderers/SW/GSVertexSW.h
+    Renderers/OpenGL/GLLoader.h
+    Renderers/OpenGL/GLState.h
+    Renderers/OpenGL/GSDeviceOGL.h
+    Renderers/OpenGL/GSRendererOGL.h
+    Renderers/OpenGL/GSShaderOGL.h
+    Renderers/OpenGL/GSTextureCacheOGL.h
+    Renderers/OpenGL/GSTextureOGL.h
+    Renderers/OpenGL/GSUniformBufferOGL.h
+    Renderers/OpenGL/GSVertexArrayOGL.h
+    Renderers/OpenCL/GSRendererCL.h
+    Window/GSCaptureDlg.h
+    Window/GSDialog.h
+    Window/GSSetting.h
+    Window/GSSettingsDlg.h
+    Window/GSWnd.h
     xbyak/xbyak.h
     xbyak/xbyak_mnemonic.h
     xbyak/xbyak_util.h
     )
 
 if(Windows)
     LIST(APPEND GSdxSources
-        GSCaptureDlg.cpp
-        GSDevice11.cpp
-        GSDevice9.cpp
-        GSDeviceDX.cpp
-        GSDialog.cpp
-        GSRendererDX11.cpp
-        GSRendererDX9.cpp
-        GSRendererDX.cpp
-        GSSettingsDlg.cpp
-        GSTexture11.cpp
-        GSTexture9.cpp
-        GSTextureCache11.cpp
-        GSTextureCache9.cpp
-        GSTextureFX11.cpp
-        GSTextureFX9.cpp
-        GSTextureFX.cpp
+        Renderers/DX11/GSDevice11.cpp
+        Renderers/DX9/GSDevice9.cpp
+        Renderers/DXCommon/GSDeviceDX.cpp
+        Window/GSDialog.cpp
+        Renderers/DX11/GSRendererDX11.cpp
+        Renderers/DX9/GSRendererDX9.cpp
+        Renderers/DXCommon/GSRendererDX.cpp
+        Renderers/DX11/GSTexture11.cpp
+        Renderers/DX9/GSTexture9.cpp
+        Renderers/DX11/GSTextureCache11.cpp
+        Renderers/DX9/GSTextureCache9.cpp
+        Renderers/DX11/GSTextureFX11.cpp
+        Renderers/DX9/GSTextureFX9.cpp
+        Renderers/DXCommon/GSTextureFX.cpp
```

Comment:
This file seems to be unused.

Reply:
Yes. The cmake files do include the Windows files, even though they are not used, and I updated them to reflect the new paths.

Reply:
The file GSTextureFX.cpp isn't included in the VS build files so it's only on CMakeList. I believe it can be purged.